nucypher/cli/commands/ursula.py

```diff
@@ -351,44 +340,53 @@ def forget(general_config, config_options, config_file):
 @group_character_options
 @option_config_file
 @option_dry_run
+@option_force
 @group_general_config
 @click.option('--interactive', '-I', help="Run interactively", is_flag=True, default=False)
 @click.option('--prometheus', help="Run the ursula prometheus exporter", is_flag=True, default=False)
 @click.option('--metrics-port', help="Run a Prometheus metrics exporter on specified HTTP port", type=NETWORK_PORT)
 @click.option("--metrics-listen-address", help="Run a prometheus metrics exporter on specified IP address", default='')
 @click.option("--metrics-prefix", help="Create metrics params with specified prefix", default="ursula")
 @click.option("--metrics-interval", help="The frequency of metrics collection", type=click.INT, default=90)
-def run(general_config, character_options, config_file, interactive, dry_run, prometheus, metrics_port, metrics_listen_address, metrics_prefix, metrics_interval):
+@click.option("--ip-checkup/--no-ip-checkup", help="Verify external IP matches configuration", default=True)
+def run(general_config, character_options, config_file, interactive, dry_run, prometheus, metrics_port,
+        metrics_listen_address, metrics_prefix, metrics_interval, force, ip_checkup):
     """Run an "Ursula" node."""
 
     worker_address = character_options.config_options.worker_address
     emitter = setup_emitter(general_config)
-    _pre_launch_warnings(emitter, dev=character_options.config_options.dev, force=None)
+    dev_mode = character_options.config_options.dev
+
+    if prometheus and not metrics_port:
+        raise click.BadOptionUsage(option_name='metrics-port',
+                                   message='--metrics-port is required when using --prometheus')
 
-    if not character_options.config_options.dev and not config_file:
+    _pre_launch_warnings(emitter, dev=dev_mode, force=None)
+
+    if not config_file and not dev_mode:
         config_file = select_config_file(emitter=emitter,
                                          checksum_address=worker_address,
                                          config_class=UrsulaConfiguration)
 
-    ursula_config, URSULA = character_options.create_character(emitter=emitter,
-                                                               config_file=config_file,
-                                                               json_ipc=general_config.json_ipc)
-
     prometheus_config: 'PrometheusMetricsConfig' = None
-    if prometheus:
+    if prometheus and not dev_mode:
         # ensure metrics port is provided
```

Comment:
We can probably remove this line now

```suggestion
```

Reply:
:heavy_check_mark: - relocated to the proper location above